src/main/java/seedu/address/model/person/Flag.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.model.person;
+
+public class Flag {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Flag should only be 'true' or 'false'.";
+    public static final String VALIDATION_REGEX = "(true|false)";
+    private boolean isFlagged;
+
+    /**
+     * Constructs a false {@code Flag}.
+     */
+    public Flag() {
+        isFlagged = false;
+    }
+
+    /**
+     * Constructs a {@code Flag} with a boolean.
+     *
+     * @param bool If the person is flagged.
+     */
+    public Flag(String bool) {
+        if (bool.equals("false")) {
+            isFlagged = false;
+        } else if (bool.toLowerCase().equals("true")) {
```

Comment:
May I know what's the reason why it requires toLowerCase here and not for false?

Reply:
Oh that is to accept case-insensitive input for "f/" prefix. 
So TRUE, True, true are all accepted.

Reply:
Does that mean false check needs to be lowercased too?

Reply:
yup i edited the isValidFlag to check after lowercased as well after the latest commit